server/templates/about_page.html

```diff
@@ -72,29 +72,78 @@ <h1>Flow is open-sourced</h1>
     </div>
   </div>
 
-  <h1>Maintainer</h1>
+  <h1>Team</h1>
   <div class="founders">
-    {% call about_blurb('/static/img/team/jeff.jpg') %}
-      <h2 class="name">Jeff Gulbronson</h2>
+    {% call about_blurb('/static/img/team/ayush.jpg') %}
+      <h2 class="name"> Ayush Kapur</h2>
       <div class="program">2B Software Engineering</div>
-      <p class="links">
-        <a class="link" target="_blank" href="http://jgulbronson.github.io">
-          <i class="icon-globe"></i> Website
-        </a>
-        <a class="link" target="_blank" href="http://ca.linkedin.com/in/jgulbronson">
-          <i class="icon-linkedin"></i> LinkedIn
-        </a>
+      <h4 class="role">Team Lead, Backend Developer</h4>
+      <p>
+        Ayush can often be found on the bike exploring the great outdoors, in a 
+        cafe with his nose in a book, or pacing excitedly while listening to a debate.
       </p>
       <p>
-        Jeff enjoys juggling and getting involved in the Waterloo Engineering
-        Society
+        He is currently enjoying the excellent weather that Vancouver has to 
+        offer while working on online services for Call of Duty: Black Ops 4 at Demonware.
       </p>
       <p>
-        He has worked at Square, enjoying all the sunny weather San Francisco
-        has to offer, and is currently enjoying all the rainy days Seattle has
-        to offer while working at Seeq.
+        Fun fact: he has eaten Alligator, and he contends that (while they may look very 
+        different) they taste exactly like chicken.
       </p>
     {% endcall %}
+
+    {% call about_blurb('/static/img/team/bob.jpg') %}
+      <h2 class="name"> Bob Wei</h2>
+      <div class="program">2B Software Engineering</div>
```

Comment:
If you want to avoid needing to update this every 4 months, you might just want to put your year :P That's how Jeff's ended up being "2B Software Engineering" until he graduated.